docs/source/links/state.md

```diff
@@ -0,0 +1,609 @@
+---
+title: apollo-link-state
+description: Manage your local data with Apollo Client
+---
+
+Managing remote data from an external API is simple with Apollo Client, but
```

Comment:
It looks like the norm in this repo is to have paragraphs that are not line-wrapped in the source, leading to single HTML paragraphs. Line breaks in the source like this appear to lead to `<br>`s and docs that don't flow with window sizes. @stubailo is this correct? maybe should be mentioned in docs-docs

Reply:
this is a result of prettier formatting my markdown, I'm going to figure out how to turn it off

Reply:
@glasser turns out that when GitHub renders markdown, it actually doesn't respect line breaks. I think we don't need to be super opinionated about how we do it, but we should probably match how GitHub renders stuff.

Reply:
Yeah I don't care what the source looks like but it's definitely weird as a reader for triple clicking to select lines instead of paragraphs, etc :)